### PR TITLE
Add support to xlC_r compiler on AIX

### PR DESCRIPTION
**Pull request by eyalg1972**
Add support for IBM xlC_r compiler for AIX

### CODE REVIEW

File: src/main/resources/com/github/maven_nar/aol.properties [hunk 1]
```diff
@@ -127,9 +127,7 @@ amd64.Windows.msvc.res.excludes=
 
 amd64.Windows.msvc.idl.compiler=midl
 amd64.Windows.msvc.idl.defines=
-amd64.Windows.msvc.idl.options=
```
Comment (ctrueden) [inline src/main/resources/com/github/maven_nar/aol.properties]: Are these removals intended? If so, I think the commit message needs to explain what this is relevant to the addition of xlC_r. Same for the removal of the `x86.Windows.gpp` lines.

**Comment by eyalg1972** [comment]
I just cleaned some duplication.... (You can still find these lines, now just in one place instead of 2)

**Comment by ctrueden** [comment]
> I just cleaned some duplication

Ah, I see, OK. Great. If you can do such things as separate commits, that would be ideal, but in the interest of pragmatism I'll merge as-is.

Thank you!

**Comment by eyalg1972** [comment]
Next time I promise a separate  commit and PR....

Eyal

**Pull request merged**